vault/logical_system.go

```diff
@@ -55,14 +55,16 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) (logical.Backen
 			Root: []string{
 				"auth/*",
 				"remount",
-				"revoke-prefix/*",
 				"audit",
 				"audit/*",
 				"raw/*",
 				"replication/primary/secondary-token",
 				"replication/reindex",
 				"rotate",
 				"config/auditing/*",
+				"lease/lookup*",
+				"lease/revoke-prefix/*",
```

Comment:
It is probably the right thing to do but this would be a change in behavior.  I just want to make sure that what we want to do.  I will note it in the changelog if we decide to go that route.